cli/cmd/cluster-apply.go

```diff
@@ -106,14 +106,20 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 			ex:         *ex,
 		}
 
-		releases := []string{"pod-checkpointer", "kube-apiserver", "kubernetes", "calico"}
+		releases := map[string]string{
+			"pod-checkpointer": "kube-system",
+			"kube-apiserver":   "kube-system",
+			"kubernetes":       "kube-system",
+			"calico":           "kube-system",
+			"lokomotive":       "lokomotive-system",
+		}
```

Comment:
I'm sorry @knrt10, my previous suggestion was not totally right. I think we should do the following:
```suggestion
              namespaces := []string{"kube-system", "lokomotive-system"}
		releases := map[string][]string{
		        "kube-system": []string{"pod-checkpointer", "kube-apiserver", "kubernetes", "calico", "lokomotive"},
		        "lokomotive-system": []string{"lokomotive"},
		 }
```

Iterating over map in Go does not have ordering guarantee and control plane components must be updated in the right order, so I think doing something like suggested should work as expected.